Translated/Dialog/Dana/ssn_014440.csv

```diff
@@ -0,0 +1,12 @@
+name_7422#0,"""Dana"""
+ssn_014440_0001#0,"""Welcome to Retem City!"""
+ssn_014440_0002#0,"""I'm Dana, one of the lookouts here!"""
+ssn_014440_0003#0,"""If the enemy attacks, you'll know from us immediately. You'll be fighting then, too."""
```

Comment:
Is this definitely "fighting then" or should it be "fighting them"

Reply:
n, but they'd mean the same thing if you wanted m instead